src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -800,15 +838,193 @@ def __try_add_error(error_list, detail):
 
     def __set_errors_json(self, error_count_by_operation, errors_by_operation):
         """ Compose the error object json to be added in 'errors' in given operation's summary """
+        code = Constants.PatchOperationTopLevelErrorCode.SUCCESS if error_count_by_operation == 0 else Constants.PatchOperationTopLevelErrorCode.ERROR
+
         message = "{0} error/s reported.".format(error_count_by_operation)
         message += " The latest {0} error/s are shared in detail. To view all errors, review this log file on the machine: {1}".format(len(errors_by_operation), self.__log_file_path) if error_count_by_operation > 0 else ""
         return {
-            "code": Constants.PatchOperationTopLevelErrorCode.SUCCESS if error_count_by_operation == 0 else Constants.PatchOperationTopLevelErrorCode.ERROR,
+            "code": code,
             "details": errors_by_operation,
             "message": message
         }
```

Comment:
Code for Warning not included?

<img width="420" alt="image" src="https://github.com/Azure/LinuxPatchExtension/assets/56841542/292ad17c-170e-425b-b6e6-39df848a5a96">

Reply:
the truncation error logic is move to pr1.1

Reply:
This code won't work well with what's in master. Please merge from master and tag me in this section to review closer. @feng-j678

Reply:
yeah, this truncation error is in pr1.1 not this pr and it has utests cover for it, I will need to make modifications when we review pr1.1

Reply:
Ok, so you've removed changes here in favor of what's in master. Sounds good. @rane-rajasi to also take a look.

Reply:
All good from me, will review the error logic part in PR1.1